templates/addthis-wrapper.html.twig

```diff
@@ -3,10 +3,10 @@
  * @file
  * Default theme implementation to display a node.
  * Available variables:
- * - element: AddThis Wrapper to be rendered
- * - element['']: Something else to render.
+ * - element: original AddThis wrapper object to reference
+ * - element['#markup']: Html markup to render for AddThis output
  *
  */
 #}
 
-{{ element.addthis_wrapper }}
\ No newline at end of file
+{{ element['#markup'] }}
```

Comment:
@damontgomery I had moved this since @doylejd & I had chatted about altering the twig slightly. I might need to chat with Doyle when he's back on Monday to see if this what he had meant. I couldn't get the element to print as {{ element }} but might be my malformed structure change.